Tests/AlicerceTests/Persistence/CoreData/CoreDataStackSetupTests.swift

```diff
@@ -1,6 +1,6 @@
 //
 //  CoreDataStackSetupTests.swift
-//  mobile_mrp_mrporter
+//  Alicerce
 //
 //  Created by André Pacheco Neves on 14/03/2017.
 //  Copyright © 2017 net-a-porter. All rights reserved.
```

Comment:
The (c) doesn't seem quite right in this file, and others... 😊

Reply:
Oops, well spotted! 😅

Reply:
I didn't notice that... 😱 Thanks!